src/plugins/auto_batch/src/compiled_model.hpp

```diff
@@ -30,6 +30,7 @@ class CompiledModel : public ov::ICompiledModel {
         std::condition_variable _cond;
         std::mutex _mutex;
         std::exception_ptr _exception_ptr;
+        bool _is_wakeup;
```

Comment:
```suggestion
       std::atomic<bool> _is_wakeup;
```
But current definition also is ok.